js/src/product-feed/review-request/review-request-issues.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { __experimentalText as Text, Button } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+
+const ReviewRequestIssues = ( { issues = [] } ) => {
+	const [ expanded, setExpanded ] = useState( false );
+
+	const toggleExpanded = () => {
+		setExpanded( ! expanded );
+		return false;
```

Comment:
Thanks. Fixed here [c4fb768](https://github.com/woocommerce/google-listings-and-ads/pull/1234/commits/c4fb76804aa318449a6f87ff7e39f38e2ed81bb5). At the beginning I as using a link instead of button which needed that.